homeassistant/components/binary_sensor/mercedesme.py

```diff
@@ -0,0 +1,109 @@
+"""
+Support for Mercedes cars with Mercedes ME.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/mercedesme/
+"""
+import logging
+import datetime
+
+from homeassistant.components.binary_sensor import (
+    BinarySensorDevice)
+
+DEPENDENCIES = ['mercedesme']
+
+DATA_MME = 'mercedesme'
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """Setup the sensor platform."""
+    if discovery_info is None:
+        return
+
+    controller = hass.data[DATA_MME]['controller']
+
+    if not controller.cars:
+        return
+
+    for car in controller.cars:
+        add_devices([BinarySensor('doorsClosed', car, controller)], True)
+        add_devices([BinarySensor('windowsClosed', car, controller)], True)
+        add_devices([BinarySensor('locked', car, controller)], True)
+        add_devices([BinarySensor('tireWarningLight', car, controller)], True)
+
+
+class BinarySensor(BinarySensorDevice):

```

Comment:
We generally include the platform name in the entity class names. Maybe `MercedesMEBinarySensor`?